default/scripting/policies/BUREAUCRACY.focs.txt

```diff
@@ -66,7 +67,7 @@ Policy
                         ]
                         OwnedBy empire = Source.Owner
                     ]
-                    tag = "needbuildings" data = (TurnsSincePolicyAdopted empire = Source.Owner name = ThisPolicy) /
+                    tag = "needbuildings" data = (TurnsSincePolicyAdopted empire = Source.Owner name = ThisPolicy + 1) /
```

Comment:
There is still a problem with this, but I have no idea how to fix it:
The message is generated the turn before, but you may finish another RegAd in the same turn and never actually lose the stability bonus.

Reply:
I suppose it doesn't need to only appear on turns when it expects the newly increased requirement to not be met...  It could instead appear on turns when the requirement will soon increase by 1, regardless of whether the empire has enough admin buildings already.

Reply:
You means something like:
  `activation = (turns + 2 % 10) == 0`
and message:
  `Next turn you'll need N, currently having M.`
I guess that could work, just have to add operator %.

Reply:
https://github.com/freeorion/freeorion/commit/cbf20c7afd52e97f4535c18228d204d0cf6f6e65

Reply:
fixing this can be a separate pull request, though...